### PR TITLE
Notebook fixes

### PR DESCRIPTION
**Pull request by andremoeller**
*Issue #, if available:*

*Description of changes:*

1. Fixed import
2. Removed matplotlib cell (seems like matplotlib was removed in EMR 5.13)
3. Use a better role ARN: get account from STS, use "SageMakerRole".

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

- [x] I have read the [CONTRIBUTING](https://github.com/aws/sagemaker-spark/blob/master/CONTRIBUTING.md) doc
- [x] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [x] I have updated the [changelog](https://github.com/aws/sagemaker-spark/blob/master/CHANGELOG.rst) with a description of my changes (if appropriate)
- [x] I have updated any necessary [documentation](https://github.com/aws/sagemaker-spark/blob/master/README.md) (if appropriate)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: examples/notebooks/zeppelin/MNIST-kmeans.json [hunk 1]
```diff
@@ -1 +1 @@
...
...ig":{"colWidth":12,"editorMode":"ace/mode/scala","results":{},"enabled":true,"editorSetting":{"language":"scala","editOnDblClick":false},"tableHide":true},"settings":{"params":{},"forms":{}},"apps":[],"jobName":"paragraph_1524718341814_-2141140640","id":"20171027-195104_2011298555","dateCreated":"2018-04-26T04:52:21+0000","status":"READY","errorMessage":"","progressUpdateIntervalMs":500,"$$hashKey":"object:112"},{"text":"%md\n\n\n","dateUpdated":"2018-04-26T05:48:38+0000","config":{"tableHide":false,"editorSetting":{"language":"markdown","editOnDblClick":true},"colWidth":12,"editorMode":"ace/mode/markdown","editorHide":true,"results":{},"enabled":true},"settings":{"params":{},"forms":{}},"apps":[],"jobName":"paragraph_1524718341814_-2141140640","id":"20171027-195210_612973143","dateCreated":"2018-04-26T04:52:21+0000","status":"READY","errorMessage":"","progressUpdateIntervalMs":500,"$$hashKey":"object:113"}],"name":"MNIST-kmeans","id":"2DBQGGZPN","angularObjects":{"2BRWU4WXC:shared_process":[],"2AM1YV5CU:shared_process":[],"2AJXGMUUJ:shared_process":[],"2ANGGHHMQ:shared_process":[],"2AKK3QQXU:shared_process":[]},"config":{"looknfeel":"default","personalizedMode":"false"},"info":{}}
```
Comment (ChoiByungWook) [inline examples/notebooks/zeppelin/MNIST-kmeans.json]: It is really difficult to see what has changed :(.

**Comment by ChoiByungWook** [comment]
Hey Andre, the commit for updating the README was committed Janaury 12th, so master will override it. Can you commit on top of it?

File: examples/notebooks/zeppelin/MNIST-kmeans.json (hunk 1 above)
Comment (andremoeller) [reply examples/notebooks/zeppelin/MNIST-kmeans.json]: Yeah, I described what changed in the description, but you'll have to describe . I'm looking at the diff and the README doesn't look like it'll be overridden -- could you elaborate?

Comment (ChoiByungWook) [reply examples/notebooks/zeppelin/MNIST-kmeans.json]: There used to be a commit, in the commit history. Not too sure what happened to it, maybe a force push? It isn't there anymore, so disregard my comment. Now you're making me wish that I took a picture of it.

**Pull request merged**